lib/Jsrt/ChakraDebug.h

```diff
@@ -538,7 +538,6 @@
             _In_ unsigned int objectHandle,
             _Out_ JsValueRef *handleObject);
 
-#ifdef _WIN32
```

Comment:
What would be non-Windows debugging story? Should `ch` use a Utf8 version? + @agarwal-sandeep